browser_patches/webkit/embedder/Playwright/win/WebKitBrowserWindow.cpp

```diff
@@ -385,20 +384,6 @@ void WebKitBrowserWindow::didNotHandleKeyEvent(WKPageRef, WKNativeEventPtr event
     PostMessage(thisWindow.m_hMainWnd, event->message, event->wParam, event->lParam);
 }
 
-void WebKitBrowserWindow::decidePolicyForNavigationAction(WKPageRef page, WKFrameRef frame, WKFrameNavigationType navigationType, WKEventModifiers modifiers, WKEventMouseButton mouseButton, WKFrameRef originatingFrame, WKURLRequestRef request, WKFramePolicyListenerRef listener, WKTypeRef userData, const void* clientInfo)
```

Comment:
Please revert this, you remove recently added code.

Reply:
I was wondering why `export webkit` added these changes. I had to reapply the `bootstrap.diff` patch after rebase to top of `main`.